source/dev/contribution_guide.rst

```diff
@@ -29,6 +30,14 @@ How to Contribute
 #. Document your code (see the `Document your Code`_ section)
 #. Make a pull request (see our :doc:`../user/git_guide` page)
 
+Bug Report
```

Comment:
Seems like this section is almost the same as for feature requests. Would it make sense to combine them? E.g. we could merge the bullet points into `Create an issue` and make this section generic, maybe `navigate to the repo, click on **Issues**.. select the appropriate template.. etc`

Reply:
Thank you for your feedback. That makes sense. Originally I was just going about adding the section per the issues.
I made some modifications. Let me know if you like them.

Reply:
Thanks, looks good!